app/javascript/components/analytics/widgets/Demographics.js

```diff
@@ -5,9 +5,9 @@ import { ResponsiveContainer, BarChart, Bar, XAxis, YAxis, CartesianGrid, Toolti
 import _ from 'lodash';
 
 const WORKFLOWS = ['Exposure', 'Isolation'];
-const AGEGROUPS = ['0-19', '20-29', '30-39', '40-49', '50-59', '60-69', '70-79', '>=80', 'FAKE_BIRTHDATE'];
-const SEXES = ['Male', 'Female', 'Unknown'];
-const ETHNICITIES = ['Hispanic or Latino', 'Not Hispanic or Latino'];
+const AGEGROUPS = ['0-19', '20-29', '30-39', '40-49', '50-59', '60-69', '70-79', '>=80', 'Missing', 'FAKE_BIRTHDATE'];
```

Comment:
I see you adding the "Missing" value to the labels, but where are we actually accounting for it in the analytics calculations? It seems like it will always be 0 when looking at the age calculations here, for example:
https://github.com/SaraAlert/SaraAlert/blob/analytics-tweaks/app/javascript/components/analytics/widgets/Demographics.js#L47

Reply:
Yes, but you are correct that Missing is never added to age in the Cache Analytics Job (it's the only one). Fixing that too.

Reply:
Yeah was just about to unresolve to say it looks like it's for Age only. Thanks!